ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_security.schema.yml

```diff
@@ -11,6 +11,26 @@ keys:
     keys:
       entropy_source:
```

Comment:
Please move testing of this key to the deprecated_vars scenario. thanks.

Reply:
Done